modules/cluster-autoscaler-about.adoc

```diff
@@ -11,7 +11,14 @@ The cluster autoscaler adjusts the size of an {product-title} cluster to meet it
 
 The cluster autoscaler increases the size of the cluster when there are pods that fail to schedule on any of the current worker nodes due to insufficient resources or when another node is necessary to meet deployment needs. The cluster autoscaler does not increase the cluster resources beyond the limits that you specify.
 
-The cluster autoscaler computes the total memory, CPU, and GPU on all nodes the cluster, even though it does not manage the control plane nodes. These values are not single-machine oriented. They are an aggregation of all the resources in the entire cluster. For example, if you set the maximum memory resource limit, the cluster autoscaler includes all the nodes in the cluster when calculating the current memory usage. That calculation is then used to determine if the cluster autoscaler has the capacity to add more worker resources.
+The cluster autoscaler computes the total 
```

Comment:
I'm not seeing this change in the OSD or ROSA preview. I tried following it down to see why it's not, but I gave up :) 

So I can't confirm for sure if this para renders okay or not w/ the ifdefs